src/transactions/TransactionUtils.cpp

```diff
@@ -497,15 +499,15 @@ getMaxAmountReceive(LedgerTxnHeader const& header, LedgerEntry const& le)
     }
     if (le.data.type() == TRUSTLINE)
     {
-        int64_t amount = 0;
-        if (isAuthorized(le))
+        auto const& tl = le.data.trustLine();
```

Comment:
again, maybe just stylistic, but using `isAuthorizedToMaintainLiabilities` keeps the code easy to understand

Reply:
I'm not sure if it makes much sense here. That line can only be hit if the ledger version check above faile, where the `AUTHORIZED_TO_MAINTAIN_LIABILITIES_FLAG` flag shouldn't even be set.